packages/uucp/uucp.1.0.0/descr

```diff
@@ -0,0 +1,11 @@
+Unicode character properties for OCaml
+Unicode version %%UNICODE_VERSION%%
```

Comment:
Yeah it's a glitch in my distribution procedure w.r.t. to `opam-publish`.  Will be fixed once I review all this in the context of assemblage.